tests/test_parser.py

```diff
@@ -30,19 +30,19 @@ def fully_parse(program: str) -> list[Structure]:
 
 def test_basic():
     assert (
-        str(fully_parse("1 1+"))
+        fully_parse("1 1+")
         == "[['none', ['number', '1']], ['none', ['number', '1']], "
         + "['none', ['general', '+']]]"
     )
 
     assert (
-        str(fully_parse("`Hello, World!`"))
+        fully_parse("`Hello, World!`")
         == "[['none', ['string', " + "'Hello, World!']]]"
```

Comment:
see above comment; this looks like it should be a list not a string representation, unless you overloaded `__eq__` for the result of `fully_parse`

Reply:
yeah, `fully_parse` returns a plain `list`

Reply:
Yeah, the tests need to be rewritten. They used to use `__repr__` to turn into stringy lists and then they'd be `eval`ed to get a proper list back or something like that. Shouldn't be terribly difficult though, I'll get on it once we merge this.